### PR TITLE
The Witness: Put progression + useful on some items.

### PR DESCRIPTION
**Pull request by NewSoupVi**
Not much to say. We discussed a bit which items in Witness fit the new definition of prog+useful, and this is the result of that.

### CODE REVIEW

**Comment by NewSoupVi** [comment]
Deciding whether I actually wanna do this sphere thing

**Comment by NewSoupVi** [comment]
Nah I don't. Changed it to be much ~~dumber~~ simpler

**Review by ScipioWright — APPROVED** [review]
Pretty clear how it's being done here, code looks fine, did not test gen

**Comment by NewSoupVi** [comment]
Oh no

**Review by Berserker66 — APPROVED** [review]

File: worlds/witness/player_items.py [hunk 1]
```diff
@@ -100,6 +100,42 @@ def __init__(self, world: "WitnessWorld", player_logic: WitnessPlayerLogic,
             self.item_data[location_name] = ItemData(None, ItemDefinition(0, ItemCategory.EVENT),
                                                      ItemClassification.progression, False)
 
+        # Determine which items should be progression + useful, if they exist in some capacity.
+        # Note: Some of these may need to be updated for the "independent symbols" PR.
+        self._proguseful_items = {
+            "Dots", "Stars", "Shapers", "Black/White Squares",
+            "Caves Shortcuts", "Caves Mountain Shortcut (Door)", "Caves Swamp Shortcut (Door)",
+            "Boat",
+            "Town Obelisk Key",  # Most checks
```
Comment (Exempt-Medic) [inline worlds/witness/player_items.py:109]: Does this account for Obelisk Sides, where well, it's just not the most checks

Comment (NewSoupVi) [reply worlds/witness/player_items.py:109]: Good point. I'll have to change that.

**Pull request merged**